cpp/cmake/modules/raft_export.cmake

```diff
@@ -164,6 +170,10 @@ function(raft_export type project_name)
     set(RAPIDS_PROJECT_FINAL_CODE_BLOCK "${${RAPIDS_FINAL_CODE_BLOCK}}")
   endif()
 
+  if(DEFINED RAPIDS_INSTALL_FILES AND NOT RAPIDS_INSTALL_FILES)
```

Comment:
Suggestion:

```cmake
set(rapids_install_files OFF)
if(DEFINED RAPIDS_INSTALL_FILES)
  set(rapids_install_files ${RAPIDS_INSTALL_FILES})
endif()
```

Afterwards you can use `NOT rapids_install_files` instead of  `NOT DEFINED RAPIDS_INSTALL_FILES`